loader_hub/web/knowledgebase_web/README.md

```diff
@@ -0,0 +1,94 @@
+# Knowledge Base Website Loader
+
+This loader is a web crawler and scraper that fetches text content from websites structed as knowledge bases. Typically these sites have a directory structure with several sections and many articles in each section. This loader crawls and finds all links that match the article path provided, and scrapes the content of each page.
```

Comment:
Maybe it'd be good to add some examples in this first bit of what knowledge bases are? I wasn't super clear the first time reading through.